src/main/java/uk/gov/ons/census/fwmt/outcomeservice/config/GatewayEventsConfig.java

```diff
@@ -20,28 +20,31 @@
   public static final String COMET_CCSSI_OUTCOME_RECEIVED = "COMET_CCSSI_OUTCOME_RECEIVED";
   public static final String COMET_CCSPL_OUTCOME_RECEIVED = "COMET_CCSPL_OUTCOME_RECEIVED";
   public static final String COMET_CESPG_OUTCOME_RECEIVED = "COMET_CESPG_OUTCOME_RECEIVED";
+  public static final String COMET_CE_OUTCOME_RECEIVED = "COMET_CE_OUTCOME_RECEIVED";
   public static final String COMET_CESPGUNITADDRESS_OUTCOME_RECEIVED = "COMET_CESPG_UNITADDRESS_OUTCOME_RECEIVED";
   public static final String COMET_CESPGSTANDALONE_OUTCOME_RECEIVED = "COMET_CESPG_STANDALONE_OUTCOME_RECEIVED";
   public static final String PREPROCESSING_CESPG_OUTCOME = "PREPROCESSING_CESPG_OUTCOME";
   public static final String PREPROCESSING_CESPGUNITADDRESS_OUTCOME = "PREPROCESSING_CESPGUNITADDRESS_OUTCOME";
   public static final String PREPROCESSING_CESPGSTANDALONE_OUTCOME = "PREPROCESSING_CESPGSTANDALONE_OUTCOME";
   public static final String PROCESSING_CESPG_OUTCOME = "PROCESSING_CESPG_OUTCOME";
+  public static final String PROCESSING_CE_OUTCOME = "PROCESSING_CE_OUTCOME";
 
   //PROPERTY_LISTING_SENT
   public static final String CCSPL_OUTCOME_SENT = "CCSPL_OUTCOME_SENT";
   public static final String CCSPL_CACHED_OUTCOME = "CCSPL_CACHED_OUTCOME";
   public static final String CCSPL_CACHED_FAILED = "CCSPL_CACHED_FAILED";
 
-  //SPG_OUTCOME
-  public static final String CESPG_OUTCOME_SENT = "CESPG_OUTCOME_SENT";
-  public static final String CESPG_ADDRESS_NOT_VALID_OUTCOME_SENT = "CESPG_ADDRESS_NOT_VALID_OUTCOME_SENT";
-  public static final String CESPG_ADDRESS_TYPE_CHANGED_OUTCOME_SENT = "CESPG_ADDRESS_TYPE_CHANGED_OUTCOME_SENT";
-  public static final String CESPG_FULFILMENT_REQUESTED_OUTCOME_SENT = "CESPG_FULFILMENT_REQUESTED_OUTCOME_SENT";
-  public static final String CESPG_EXTRAORDINARY_REFUSAL_RECEIVED_OUTCOME_SENT = "CESPG_EXTRAORDINARY_REFUSAL_RECEIVED_OUTCOME_SENT";
-  public static final String CESPG_HARD_REFUSAL_RECEIVED_OUTCOME_SENT = "CESPG_HARD_REFUSAL_RECEIVED_OUTCOME_SENT";
-  public static final String CESPG_QUESTIONNAIRE_LINKED_OUTCOME_SENT = "CESPG_QUESTIONNAIRE_LINKED_OUTCOME_SENT";
-  public static final String CESPG_NEW_ADDRESS_REPORTED_OUTCOME_SENT = "CESPG_NEW_ADDRESS_REPORTED_OUTCOME_SENT";
-  public static final String CESPG_NEW_UNIT_ADDRESS_OUTCOME_SENT = "CESPG_NEW_UNIT_ADDRESS_OUTCOME_SENT";
+  //OUTCOMES
+  public static final String OUTCOME_SENT = "OUTCOME_SENT";
```

Comment:
youll be happy to knpw that the knew incarnation of events will have all this registering of events removed